fabrics.go

```diff
@@ -26,9 +27,9 @@ type FabricVLAN struct {
 
 type ListFabricVLANsInput struct{}
 
-func (client *FabricsClient) ListFabricVLANs(*ListFabricVLANsInput) ([]*FabricVLAN, error) {
+func (client *FabricsClient) ListFabricVLANs(ctx context.Context, _ *ListFabricVLANsInput) ([]*FabricVLAN, error) {
```

Comment:
Just for my own education, and I realize this isn't a change in this PR: do we have these unused parameters (in various places) as placeholders so that we can change the implementations without breaking the API contract? Given that the consumer would need to swap out the empty/placeholder input struct for a populated one, what does this give us? I may just be missing something here.